catroid/src/org/catrobat/catroid/utils/ImageEditing.java

```diff
@@ -180,6 +182,27 @@ private static double calculateScaleFactor(int originalWidth, int originalHeight
 		}
 	}
 
+	public static Vector2 calculateXYScaleFactorToScreenSize(int resourceId, Context context) {
```

Comment:
Rename calculateXYScaleFactorToScreenSize or calculateScaleFactor to be consistent.